pkg/controller/workspacerouting/solvers/oauth_proxy_container.go

```diff
@@ -0,0 +1,117 @@
+//
+// Copyright (c) 2019-2020 Red Hat, Inc.
+// This program and the accompanying materials are made
+// available under the terms of the Eclipse Public License 2.0
+// which is available at https://www.eclipse.org/legal/epl-2.0/
+//
+// SPDX-License-Identifier: EPL-2.0
+//
+// Contributors:
+//   Red Hat, Inc. - initial API and implementation
+//
+
+package solvers
+
+import (
+	"fmt"
+	"github.com/che-incubator/che-workspace-operator/pkg/apis/workspace/v1alpha1"
+	"github.com/che-incubator/che-workspace-operator/pkg/common"
+	"github.com/che-incubator/che-workspace-operator/pkg/config"
+	corev1 "k8s.io/api/core/v1"
+	"strconv"
+)
+
+const proxyServiceAcctAnnotationKeyFmt string = "serviceaccounts.openshift.io/oauth-redirectreference.%s-%s"
+const proxyServiceAcctAnnotationValueFmt string = `{"kind":"OAuthRedirectReference","apiVersion":"v1","reference":{"kind":"Route","name":"%s"}}`
+
+var openShiftProxySARFmt = `{"namespace": "%s", "resource": "pods", "name": "%s", "verb": "exec"}`
+
+func getProxyPodAdditions(proxyEndpoints map[string]proxyEndpoint, meta WorkspaceMetadata) *v1alpha1.PodAdditions {
+	var proxyContainers []corev1.Container
+	for _, proxyEndpoint := range proxyEndpoints {
+		proxyContainers = append(proxyContainers, getProxyContainerForEndpoint(proxyEndpoint, meta))
+	}
+	proxyVolumes := getProxyVolumes(proxyContainers)
+	serviceAcctAnnotations := getProxyServiceAcctAnnotations(proxyEndpoints, meta)
```

Comment:
```suggestion
	serviceAccAnnotations := getProxyServiceAccAnnotations(proxyEndpoints, meta)
```

Reply:
I prefer Acct as it's a more common abbreviation of account

Reply:
OK